webservices/resources/decorators.py

```diff
@@ -0,0 +1,36 @@
+from functools import wraps
+from time import time
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+def timing(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        LOGGER.info(
+            '*' * 10 + 'STEP {} starts to work'.format(func.__name__) + '*' * 10 + '\n')  # be compatible to Python2
+        ts = time()
+        res = func(*args, **kwargs)
+        te = time()
+        LOGGER.info('STEP {} took {} seconds'.format(func.__name__, round(te - ts, 2)) + '\n')
+        return res
+    return wrapper
+
+# TODO: check this link to have an updated version of query print:
+# https://stackoverflow.com/questions/5631078/sqlalchemy-print-the-actual-query
+def print_query(func):
```

Comment:
This @print_query function doesn't include the parameters, so I'm not sure it adds value - This feature needs more work and should probably go in another ticket for testing. To include parameters, you can do this, but it won't work for dates:
```
from sqlalchemy.dialects import postgresql

print(str(query.statement.compile(
    dialect=postgresql.dialect(),
    compile_kwargs={"literal_binds": True})))
```